docs/Logging.md

```diff
@@ -71,6 +71,33 @@ const fastify = require('fastify')({
   }
 })
 ```
+For example, the response payload can be logged using the approach below
+
+```js
+const fastify = require('fastify')({
+  logger: {
+    prettyPrint: true,
+    serializers: {
+      res(res) {
+        // the default
+        return {
+          statusCode: res.statusCode
+        }
+      },
+      req(req) {
+        return {
+          method: req.method,
+          url: req.url,
+          path: req.path,
+          parameters: req.parameters,
+          body: req.body,
```

Comment:
```suggestion
          // Including the body and headers in the log could be in violation 
          // of privacy laws, e.g. GDPR. You should use the "redact" option to
          // remove sensitive fields. It could also leak authentication data in
          // the logs.
          body: req.body,
```